_redirects

```diff
@@ -2,3 +2,31 @@ http://rmarkdown.rstudio.com/*   https://rmarkdown.rstudio.com/:splat  301!
 https://rmarkdown.netlify.com/*  https://rmarkdown.rstudio.com/:splat  301!
 http://rmarkdown.netlify.com/*   https://rmarkdown.rstudio.com/:splat  301!
 /flexdashboard    /flexdashboard/   301!
+
+/r_notebooks.html  https://bookdown.org/yihui/rmarkdown/notebook.html  301!
+/html_document_format.html https://bookdown.org/yihui/rmarkdown/html-document.html  301!
+/pdf_document_format.html  https://bookdown.org/yihui/rmarkdown/pdf-document.html  301!
```

Comment:
Levels of section headers.

Figure/table captions.

Cross-references of sections.